### PR TITLE
add empty correlation skeleton

### PR DESCRIPTION
**Pull request by asizemore**
Resolves task in https://github.com/VEuPathDB/web-monorepo/issues/512

Adds the skeleton of a correlation bipartite network viz so that the frontend can actually see a visualization. The goal is to really just let the frontend have the endpoint, but as a byproduct we've gone ahead and set up some of the plugin skeleton.

### CODE REVIEW

File: schema/library.raml [hunk 1]
```diff
@@ -373,7 +372,7 @@ types:
       isMergeKey: boolean
       isMultiValued: boolean
       imputeZero: boolean
-      hasStudyDependentVocabulary?: boolean
+      hasStudySpecificVocabulary?: boolean
```
Comment (d-callan) [inline schema/library.raml:375]: whats happened here? this isnt right..

File: schema/library.raml [hunk 2]
```diff
@@ -70,7 +70,6 @@ types:
       displayRangeMax?: any
       vocabulary?: string[]
       imputeZero: boolean
-      hasStudyDependentVocabulary?: boolean
```
Comment (d-callan) [inline schema/library.raml:73]: boo

File: schema/library.raml [hunk 3]
```diff
@@ -777,6 +776,12 @@ types:
       overlayVariable:
         type: VariableSpec
         required: false
+  CorrelationAssayMetadataBipartitenetworkPostRequest:
+    type: DataPluginRequestBase
+    properties:
+      computeConfig: EmptyDataPluginSpec
+      config: EmptyDataPluginSpec
+  CorrelationAssayMetadataStatsResponse: DifferentialAbundanceStatsResponse
```
Comment (d-callan) [inline schema/library.raml:784]: do we not know what the response will look like yet?

File: schema/library.raml (hunk 1 above)
Comment (asizemore) [reply schema/library.raml:375]: i ran `make raml-gen-code` which made this change. Is the original name the correct one? I can remove this change from the PR

File: schema/library.raml (hunk 3 above)
Comment (asizemore) [reply schema/library.raml:784]: we have a plan but since it doesn't matter for the frontend stub i didn't add it yet

**Comment by asizemore** [comment]
@d-callan do you have any concerns that I need to address before merging? The frontend for correlation is now up but without this piece no one can see. 
I'll be back at the data service in a few days to turn this skeleton into a real plugin

**Comment by d-callan** [comment]
I'd like to know why that new prop is wrong and fix that..

**Comment by asizemore** [comment]
Do you mean `hasStudySpecificVocabulary`? Fixed in the latest commit. I'll resolve the comment above about it so that it's more clear

**Review by d-callan — APPROVED** [review]

**Pull request merged**